<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EKFConfig</class>
 <widget class="QWidget" name="EKFConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EKF Attitude Noise</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EKF Translation Noise</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EKF Linear Velocity Noise</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EKF Angular Velocity Noise</string>
   </property>
  </widget>
  <widget class="QLabel" name="QNoise">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>QNoise</string>
   </property>
  </widget>
  <widget class="QSlider" name="wnoise_slider">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="WNoise">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>WNoise</string>
   </property>
  </widget>
  <widget class="QLabel" name="VNoise">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>VNoise</string>
   </property>
  </widget>
  <widget class="QSlider" name="qnoise_slider">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="TNoise">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TNoise</string>
   </property>
  </widget>
  <widget class="QSlider" name="tnoise_slider">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="vnoise_slider">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QtCharts::QChartView" name="chart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>621</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="YawCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>73</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="PitchCheckBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>73</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="RollCheckBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>73</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="XCheckBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>73</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="YCheckBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>73</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ZCheckBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>73</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <zorder>YCheckBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>QNoise</zorder>
  <zorder>wnoise_slider</zorder>
  <zorder>WNoise</zorder>
  <zorder>VNoise</zorder>
  <zorder>qnoise_slider</zorder>
  <zorder>TNoise</zorder>
  <zorder>tnoise_slider</zorder>
  <zorder>vnoise_slider</zorder>
  <zorder>chart</zorder>
  <zorder>YawCheckBox</zorder>
  <zorder>PitchCheckBox</zorder>
  <zorder>RollCheckBox</zorder>
  <zorder>XCheckBox</zorder>
  <zorder>ZCheckBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
